packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -29,7 +29,6 @@ class BuildApkCommand extends BuildSubCommand {
     addDartObfuscationOption();
     usesDartDefineOption();
     usesExtraDartFlagOptions(verboseHelp: verboseHelp);
-    addBundleSkSLPathOption(hide: !verboseHelp);
```

Comment:
Was this option marked as deprecated before? If not, should we hold off on removing them completely and print an error when users try and provide them for now?